### PR TITLE
YIELD ONE amp-ad support

### PR DESCRIPTION
**Pull request by Yuka222**
Could you merge YIELDONE amp-ad support into ampproject/amphtml master branch?
We already signed CLA under the name of Platform One Inc.

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -176,8 +177,12 @@ export const adPreconnect = {
     'https://mads.at.atwola.com',
     'https://aka-cdn.adtechus.com',
   ],
+  yieldone: [
+    'https://img.ak.impact-ad.jp'
```
Comment (lannka) [inline ads/_config.js]: host listed above is not needed.

**Comment by lannka** [comment]
@Yuka222 did you get it working locally? The ad in ads.amp.html doesn't load.

**Comment by Yuka222** [comment]
@lannka Yes, I checked the ads in local.

**Comment by lannka** [comment]
@Yuka222 it's not working. did you run `gulp` locally and verify via `http://localhost:8000/examples.build/ads.amp.max.html` ?

File: examples/ads.amp.html [hunk 2]
```diff
@@ -494,5 +494,14 @@
       data-zid="1077330">
   </amp-ad>
 
+  <h2>YIELD ONE</h2>
+  <amp-ad
+   width="320"
+   height="50"
+   type="yieldone"
+   yieldoneParam.pubid="0214"
```
Comment (lannka) [inline examples/ads.amp.html]: you need `data-` prefix: `data-pubid="0214"`
same below

**Comment by lannka** [comment]
Working now! Thanks.
Please resolve the merge conflict.
Otherwise LGTM.

**Comment by Yuka222** [comment]
Hi @lannka does the CLA not approved yet?  Also, Travis failed should I change something?

**Comment by lannka** [comment]
@Yuka222 Travis failed with the error shown below:
```
SyntaxError: /home/travis/build/ampproject/amphtml/ads/_config.js: Unexpected token (213:2)
  211 |     'https://aka-cdn.adtechus.com',
  212 |   ]
> 213 |   zergnet: [
      |   ^
  214 |     'https://www.zergnet.com',
  215 |     'https://zergnet.com',
  216 |   ],
```
Seems you're missing a ''," at the end.
You can always click on the "Details" link on the failed travis to see the error logs.

CLA is auto-approved immediately after you click through the links. Please follow the instruction:
> Once you've signed, please reply here (e.g. I signed it!) and we'll verify. 

You can just reply as a comment.

File: 3p/integration.js [hunk 3]
```diff
@@ -151,6 +152,11 @@ register('webediads', webediads);
 register('weborama-display', weboramaDisplay);
 register('widespace', widespace);
 register('yahoojp', yahoojp);
+register('chargeads', chargeads);
```
Comment (lannka) [inline 3p/integration.js]: seems like a bad merge. could you revert irrelevant changes?

**Comment by Yuka222** [comment]
I signed it! Our company name is Platform One Inc.

**Comment by Yuka222** [comment]
Finally Travis passed!! Regarding the CLA I am still wondering how should I do this. If it doesn't work please let me know more detail.

**Comment by lannka** [comment]
No worry. I checked CLA list, Platform One Inc. is on it. @dvoytenko could you help to force merge?

**Comment by erwinmombay** [comment]
force merging per @lannka

**Comment by Yuka222** [comment]
Thank you for your support @lannka @erwinmombay !!

**Comment by Yuka222** [comment]
Hi @lannka  I have a question about press release. We would like to publish press release about YIELDONE supports AMP-ad but we are not sure who should we contact regarding this approval on Google side. Could you kindly connect to someone may in charge of PR about AMP in Japan?

**Comment by lannka** [comment]
> YIELDONE supports AMP-ad but we are not sure who should we contact regarding this approval on Google side. Could you kindly connect to someone may in charge of PR about AMP in Japan?

@rudygalfi do you have any idea?

**Comment by Yuka222** [comment]
@lannka We could contact Google Japan side so resolving this. Thank you for your help!

**Pull request merged**